packages/lake/src/components/LakeTooltip.tsx

```diff
@@ -133,6 +134,7 @@ export const LakeTooltip = memo(
         width,
         togglableOnFocus = false,
         containerStyle,
+        showTooltip = false,
```

Comment:
I'd name it `active` with a default as `true`, WDYT?

Reply:
I like `disabled` (with default at `false`) more. For consistency, and it's actually always weird when default is `true`. But @sandrine-ds could add a `const active = !disabled` in the component code if she want.

Reply:
agree, I think default props to true isn't intuitive